backend/src/controllers/question.js

```diff
@@ -1,19 +1,18 @@
-const express = require('express')
-const router = express.Router({ mergeParams: true })
+const qdb = require('../models/db-questions.js')
 
-/* Get the question data for a specific question id */
-router.get('/:id', function (req, res) {
-    // let qid = req.params.id;
+/* GET question data. */
+exports.getQuestion = function (req, res) {
```

Comment:
given we don't use the req object directly you could rewrite like this anytime this occurs.

```js
exports.getQuestion = function ({params}, res) { // <-- this is 'destructuring'
    // do stuff with param
}
```